diagnostic.md

```diff
@@ -34,5 +46,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{each contact.phone as |phone|}}
+      {{my-contact phone=phone}}
```

Comment:
This line should be:
`{{my-phone phone=phone}}`

It seems clear that you're understanding the concept, though.

.5/1